### ISSUE TITLE
BatchNorm does not expose bias,weight,running_mean,running_var 

### ISSUE THREAD

**Issue by Al-Dyachkov** [dotnet/TorchSharp#476]
Current BatchNorm1/2/3d is missing ability to get/set bias and weight and get running stats PyTorch has.

**Comment by Al-Dyachkov** [comment]
This was fast, thanks. 👍

**Comment by NiklasGustafsson** [comment]
Not on NuGet yet, though
